packages/libs/eda/src/lib/core/components/computations/plugins/differentialabundance.tsx

```diff
@@ -164,7 +170,15 @@ function DifferentialAbundanceConfigDescriptionComponent({
 // Include available methods in this array.
 // 10/10/23 - decided to only release Maaslin for the first roll-out. DESeq is still available
 // and we're poised to release it in the future.
-const DIFFERENTIAL_ABUNDANCE_METHODS = ['Maaslin']; // + 'DESeq' in the future
+type DifferentialAbundanceMethodCitations = { Maaslin: string };
+const DIFFERENTIAL_ABUNDANCE_METHOD_CITATIONS: DifferentialAbundanceMethodCitations =
+  {
+    Maaslin:
+      '<a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1009442">(Mallick et al., 2021)</a>',
+  }; // + deseq paper in the future
```

Comment:
You can use `as const` to make typescript infer a `Record` type using the defined properties as string literals. So, no need for the type `DifferentialAbundanceMethodCitations`.
```suggestion
const DIFFERENTIAL_ABUNDANCE_METHOD_CITATIONS =
  {
    Maaslin:
      '<a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1009442">(Mallick et al., 2021)</a>',
  } as const; // + deseq paper in the future
```